openapi/openapi_client/__init__.py

```diff
@@ -0,0 +1,55 @@
+# coding: utf-8
+
+# flake8: noqa
+
+"""
+    Jellyfish Media Server
+
+    No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
```

Comment:
If it's an easy fix, you could copy over the basic Jellyfish description (but that's not a big issue)

Reply:
done!